### DISCUSSION TITLE
Adding OpenUtau to Flathub

### DISCUSSION THREAD

**Discussion by asumbek** [stakira/OpenUtau#1277] [General]
Hello, I would like to ask if I'm allowed to add OpenUtau into [Flathub](https://flathub.org)?

**Comment by MaximusWolfie** [comment]
I personally believe that would be Stakira's decision to make

**Reply by asumbek** [reply]
But would they accept if the community port OpenUtau to Flathub?

**Reply by MaximusWolfie** [reply]
probably, but you should ask on the Discord server about it first.

**Comment by oxygen-dioxide** [comment]
> Hello, I would like to ask if I'm allowed to add OpenUtau into [Flathub](https://flathub.org)?

Of course you can, because OpenUtau is open source software under MIT license.

**Comment by asumbek** [comment]
@stakira confirmed in Discord server that I'm allowed to add it as long as it works.
